wandb/integration/keras/callbacks/tables_builder.py

```diff
@@ -107,14 +117,16 @@ def __init__(
 
         self.data_table_columns = data_table_columns
         self.pred_table_columns = pred_table_columns
+        self.in_memory = in_memory
```

Comment:
This is a good workaround, but maybe add a `wandb.termwarn` if `in_memory = True` informing the user that the data to be logged at the end of training is stored in memory and that this should not be used for logging large validations sets as it can consume system RAM.